gitpod-io/gitpod
# [registry-facade] Add supervisor ref to ImageSpec

## Description
This PR adds `supervisor_ref` to `ImageSpec` in the `registry-facade` that allows to set the supervisor image to use for a workspace start. It is meant to replace the supervisor as a static layer in a next step.

## Related Issue(s)

It's one step for #6512.

## How to test
- Verify that this change does not break something, esp. that it is still possible to start workspaces.
- Do you want to see the big picture in action? This branch has the missing piece that uses the new `supervisor_ref` field: 
   - https://github.com/gitpod-io/gitpod/tree/clu/deploy-supervisor-6512
   - https://clu-deploy-supervisor-6512.staging.gitpod-dev.com/workspaces

## Release Notes

```release-note
NONE
```
